packages/shared/src/utils/loader/lazy-load.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import Loadable from 'react-loadable';
 
-export const makeLazyLoader = (importFn, loaderFn) => component_name =>
+export type TFunction = () => any;
+
+export const makeLazyLoader = (importFn: TFunction, loaderFn: TFunction) => (component_name?: string) =>
```

Comment:
```suggestion
export const makeLazyLoader = (importFn: () => Promise<unknown>, loaderFn: () => JSX.Element) => (component_name?: string) =>
```
@niloo-fs Base on the usage of `importFn` it seems like the type is `() => Promise<JSX.Element>` but I'm not sure, So let’s go with `unknown` if you are not sure too 😅